client/src/app/_services/auth.service.ts

```diff
@@ -125,46 +90,48 @@ export class AuthService {
     }));
     this.ngxPermissionsService.flushPermissions();
     this.ngxRolesService.flushRoles();
-    if (localStorage.getItem('currentUser')) {
-      localStorage.removeItem('currentUser');
-    }
+    if (localStorage.getItem('currentUser')) localStorage.removeItem('currentUser');
+    if (sessionStorage.getItem('currentUser')) sessionStorage.removeItem('currentUser');
     this.router.navigate(['/']);
   }
 
-  private saveUser(jwt): void {
-    localStorage.setItem('currentUser', JSON.stringify(jwt));
+  private saveUser(jwt, rememberMe): void {
+    if (rememberMe) localStorage.setItem('currentUser', JSON.stringify(jwt));
```

Comment:
Heere you could add a boolean to know if its local or session

Reply:
`rememberMe` already play this role, is'nt it ?

Reply:
Yes, I forgot why I said that... I think I was thinking about using the field `longTime` (cf. upper comment) instead of using the rememberMe bool. It's not really important but can prevent sending a boolean to every functions called